packages/odyssey-storybook/src/components/Text/Text.mdx

```diff
@@ -1,7 +1,9 @@
-import { Canvas, Story, ArgsTable } from "@storybook/addon-docs";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
 import { theme } from "../../../../odyssey-react/src/components/Text/Text.theme";
 import { ThemeTable } from "../../../.storybook/components";
-import { Text } from "../../../../odyssey-react/src";
+
+<Meta anchor />
+import {Text} from "../../../../odyssey-react/src";
```

Comment:
can we move this back up with the other imports?

Reply:
Weird, `prettier` moved this down. Let me get it back up there and try again.

Reply:
Yeah, all imports have to be grouped together in JS. Dunno about MDX, but I'd assume it follows the same rules. Surprised it works at all. Probably hoisting.